src/app/treasure-box/page.tsx

```diff
@@ -26,6 +26,7 @@ export default function TreasureBox() {
   const isClient = useClientCheck();
   const { address } = useAccount();
   const { data: userPublicProfile } = useCBProfile({ address });
+
   const { attackBox } = useMutateTreasureBox();
 
   const { data: treasureBox, isLoading } = useTreasureBox({ gameId: GAME_ID });
```

Comment:
nit: attackBox may have a loading state, consider an alias for `isLoading` from useTreasureBox